plugin/etcd/msg/service.go

```diff
@@ -34,6 +34,11 @@ type Service struct {
 
 	// Etcd key where we found this service and ignored from json un-/marshalling
 	Key string `json:"-"`
+
```

Comment:
your PR prompted  me to look into this struct and we should not bloat it anymore.

If we need this we should add a pointer to a CAA record struct and figure how to get the info out from etcd.
Also see #2371